Simplenote/ContentRecoveryManager.swift

```diff
@@ -0,0 +1,74 @@
+import Foundation
+import UniformTypeIdentifiers
+import CoreData
+
+public class ContentRecoveryManager {
+    private let fileManager: FileManager
+
+    public init(fileManager: FileManager = .default) {
+        self.fileManager = fileManager
+    }
+
+    private func createRecoveryDirIfNeeded() {
+        guard !fileManager.directoryExistsAtURL(fileManager.recoveryDirectoryURL) else {
+            return
+        }
+
+        try? fileManager.createDirectory(at: fileManager.recoveryDirectoryURL, withIntermediateDirectories: true)
```

Comment:
This could probably live within a FileManager extension, so that you don't need to invoke it from `archiveContent` or `prepareRecoveredNoteContentIfNeeded`